workspaces/rbac/plugins/rbac-backend/package.json

```diff
@@ -34,8 +34,8 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
-    "@backstage-community/plugin-rbac-common": "^1.12.1",
-    "@backstage-community/plugin-rbac-node": "^1.8.1",
+    "@backstage-community/plugin-rbac-common": "workspace:^",
```

Comment:
@christoph-jerolimov Curious, is there a CLI (`yarn`?) way of doing this or was this manually changed?

Reply:
@planeiii-te I changed this manually